### PR TITLE
Db mk 2149 refactor routes

### PR DESCRIPTION
**Pull request by kimminahm**
Refactor routes to use FABS prefix rather than the detachedprefix, went and hunted down what i think are all the detached links

### CODE REVIEW

File: src/js/containers/help/HelpContainer.jsx [hunk 1]
```diff
@@ -49,22 +49,23 @@ class HelpPageContainer extends React.Component {
     }
 
     render() {
-       if (this.props.route.path == 'help' || this.props.route.path == 'detachedHelp') {
+        let currentRoute = this.props.route.path.toLowerCase();
+       if (currentRoute == 'help' || currentRoute == 'FABSHelp'.toLowerCase()) {
             return <HelpPage {...this.props} helpOnly={this.state.helpOnly} type={this.state.type} />;
        }
-       else if (this.props.route.path == 'resources' || this.props.route.path == 'detachedResources') {
+       else if (currentRoute == 'resources' || currentRoute == 'FABSResources'.toLowerCase()) {
```
Comment (kimminahm) [inline src/js/containers/help/HelpContainer.jsx]: your right....

File: src/js/containers/router/RouterRoutes.jsx [hunk 2]
```diff
@@ -267,140 +228,118 @@ const routeDefinitions = {
...
+    }
+}
+
+// defining the routes outside of the component because React Router cannot handle state/prop changes that Redux causes
+const routeDefinitions = {
+    path: '/',
+    indexRoute: {
+        onEnter: checkUserPermissions,
+        component: LandingPage,
+        type: kGlobalConstants.STAGING ? 'home' : 'dabs'
```
Comment (kimminahm) [inline src/js/containers/router/RouterRoutes.jsx]: oversight

File: src/js/containers/router/RouterRoutes.jsx [hunk 3]
```diff
@@ -267,140 +228,118 @@ const routeDefinitions = {
...
             getComponent(nextState, cb) {
                 require.ensure([], (require) => {
-                    cb(null, require('../../containers/help/HelpContainer.jsx').default)
+                    cb(null, require('../../components/dashboard/DashboardPage.jsx').default)
                 });
             },
-            type: 'fabs'
-        },
-        {
-            path: '*',
```
Comment (kimminahm) [inline src/js/containers/router/RouterRoutes.jsx]: reinserted

File: src/js/containers/help/HelpContainer.jsx [hunk 4]
```diff
@@ -50,22 +50,22 @@ class HelpPageContainer extends React.Component {
 
     render() {
         let currentRoute = this.props.route.path.toLowerCase();
-       if (currentRoute == 'help' || currentRoute == 'FABSHelp'.toLowerCase()) {
+       if (currentRoute == 'help' || currentRoute == 'fabsHelp') {
```
Comment (alburde1) [inline src/js/containers/help/HelpContainer.jsx]: This isn't lowercase

**Pull request merged**